index.js

```diff
@@ -0,0 +1,102 @@
+const playersData = "http://localhost:3000/players"
+const playerContainer = document.querySelector(".player-container")
+
+// DELIVERABLE #1
+
+// get player data
+fetch(playersData)
+    .then(response => response.json())
+    .then(data => buildPlayerElements(data))
+
+// build all players
+function buildPlayerElements(data) {
+    data.forEach(buildPlayer)
+}
```

Comment:
Same thing with these variable names - try to be more descriptive than `data`. It helps to include the data type in the variable name, particularly in a dynamically typed language like Javascript. It'll make it easier to come back to this code and see what it's doing.